### PR TITLE
Create Template for new Workloads

### PR DESCRIPTION
**Pull request by zehicle**
this uses a very simple SED approach to creating the workload.

0. Create a new repo for your workload
1. Edit the template.sed file to update the patterns.  Extend if needed.
2. Use the ./rename.rb file to remap the template names into your workload
3. Rename will copy files into your workload directory (based on the sed rules)

### CODE REVIEW

**Comment by zehicle** [comment]
@galthaus or @VictorLowther please review

File: noop/roles/ocb-template-ready/role-template.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (galthaus) [inline noop/roles/ocb-template-ready/role-template.json]: There aren't needed anymore and should be deleted.  Use attrib defaults.

File: crowbar.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2014, Dell
+# Copyright 2014, OCBTemplateAuthor
```
Comment (galthaus) [inline crowbar.yml]: 2015, nit

File: crowbar.yml [hunk 3]
```diff
@@ -13,26 +13,28 @@
...
 
 os_support:
   - ubuntu-12.04
-  - ubuntu-13.04
-  - redhat-6.5
-  - centos-6.5
-  - opensuse-13.1
-  - fedora-19
+  - centos-6.6
+  - centos-7.0.1406
```
Comment (galthaus) [inline crowbar.yml]: centos-7.1.1503

**Pull request merged**